garnet/src/WheelSliderControllerSample.js

```diff
@@ -2,40 +2,27 @@ require('garnet');
 
 var
 	kind = require('enyo/kind'),
-	IconButton = require('garnet/IconButton'),
-	Panel = require('garnet/Panel'),
 	WheelSliderController = require('garnet/WheelSliderController');
 
-var WheelSliderControllerPanel = kind({
-	name: 'g.sample.WheelSliderController',
-	kind: Panel,
+var WheelSliderPanel = kind({
+	name: 'g.sample.WheelSliderPanel',
+	kind: WheelSliderController,
 	events: {
 		onResult: ''
 	},
-	classes: 'g-common-width-height-fit g-layout-absolute-wrapper',
+	minimumValue: 0,
+	maximumValue: 100,
+	stepValue: 5,
+	value: 50,
```

Comment:
It's better to define minimumValue, maximumValue, stepValue and value in not WheeSliderPanel but g.sample.WheelSliderController